thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/auth/AuthCookieSerializer.java

```diff
@@ -0,0 +1,40 @@
+package com.linkedin.thirdeye.auth;
+
+import java.security.Key;
+import javax.crypto.Cipher;
+import org.apache.commons.codec.binary.Base64;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.codehaus.jackson.type.TypeReference;
+
+
+public class AuthCookieSerializer {
+  final Key aesKey;
+  final ObjectMapper mapper;
```

Comment:
ObjectMapper is a heavy object but thread-safe. Make it final static and be shared across instances.

Reply:
AuthCookieSerializer is instantiated by the dashboard app, and dependencies are injected.

Reply:
i've changed the constructor to clarify this